content/docs/custom-plugins/index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Installing Custom Plugins
+lastUpdated: '2021-03-17T12:11:00.0Z'
+description: How to add a custom Backstage plugin to Roadie
+---
+
+## Introduction
+
+Your plugins can be published via npm or yarn, like publishing a normal package. We will provide you a unique scope for
+your plugins, and we host a dedicated npm registry which you can publish to.
+
+### Prerequisites
```

Comment:
My suggested heading structure is this:

```plain
## Introduction

## Prerequisites

## Pushing the plugins to Roadie

### Step 1: Get credentials

### Step 2: Configure NPM

### Step 3: Publishing

## Conclusion
```

The automatically generated table of contents will include any heading which is h2. and ignore h3., h4. etc. So the table of contents will be:

```plain
1. Introduction
2. Prerequisites
3. Pushing the plugins to Roadie
4. Conclusion
```

My inspiration for this is the [Hasura docs](https://hasura.io/docs/latest/graphql/core/remote-schemas/adding-schema.html). The main difference being that we don't have so many granular pages yet (or a good way to navigate between them) so it makes sense to consolidate documentation pages more than Hasura have done.

Reply:
This is a very short document with a few steps :thinking: What should I say in the conclusion section?

Reply:
I would just summarize what has happened. Something like

> Once the steps above have been completed successfully and you have been notified that your plugin is ready to use, you should be able to log into your Backstage instance and use the plugin just like any other plugin. 

> You can update your plugin at will and the changes will appear in Backstage. The form is only required when installing a new plugin for the first time.